packages/server/src/routes/me/editMember.ts

```diff
@@ -0,0 +1,59 @@
+import { CollectionReference, Firestore } from "@google-cloud/firestore";
+
+import { EditMemberApiEndpoint } from "@raha/api-shared/dist/routes/me/definitions";
+import { firestore } from "firebase-admin";
+import { OperationType } from "@raha/api-shared/dist/models/Operation";
+import { OperationApiResponseBody } from "@raha/api-shared/dist/routes/ApiEndpoint/ApiResponse";
+import { MissingParamsError } from "@raha/api-shared/dist/errors/RahaApiError/MissingParamsError";
+
+import { createApiRoute } from "..";
+
+export const editMember = (
+  db: Firestore,
+  members: CollectionReference,
+  operations: CollectionReference
+) =>
+  createApiRoute<EditMemberApiEndpoint>(async (call, loggedInMemberToken) => {
+    const newOperationReference = await db.runTransaction(async transaction => {
+      const loggedInUid = loggedInMemberToken.uid;
+      const loggedInMember = await transaction.get(members.doc(loggedInUid));
+
+      const { full_name, username } = call.body;
+
+      if (!full_name && !username) {
```

Comment:
Should this be || ? Isn't it bad right now if either is missing? Or is this in anticipation for the edit username function

Reply:
Yeah.. I didn't particularly see a reason to prevent it... the connection between username and full name is pretty arbitrary right now, I think. I don't mind either way though.. up to you.